Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -604,18 +364,48 @@ span.bocListEditableCell .validationErrorMarker[aria-label]:not([title])::after
   box-sizing: border-box;
 }
 
-span.bocListEditableCell .validationErrorMarker[aria-label]:not([title]):focus::after,
-span.bocListEditableCell .validationErrorMarker[aria-label]:not([title]):hover::after
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell .validationErrorMarker[aria-label]:not([title]):focus::after,
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell .validationErrorMarker[aria-label]:not([title]):hover::after
 {
   visibility: visible;
   transition-delay: 0.7s;
 }
+/*endregion*/
 
-span.bocListEditableCell .control > *
+/*region Validation layout*/
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell
 {
-  width: 100%;
-  vertical-align: middle;
+  display: inline-grid;
+  grid-template-columns: auto auto;
+  grid-template-rows: auto auto;
+}
+
+div.bocList div.bocListTableBlock div.bocListTableContainer table.bocListTable span.bocListEditableCell .control
```

Comment:
quick demo of this please

Reply:
Default edit:
![image](https://user-images.githubusercontent.com/40357550/132844782-8f6ecfc5-52c2-4a97-899f-5ef7c59f015f.png)

Error:
![image](https://user-images.githubusercontent.com/40357550/132844867-9cee023a-9ccb-4d02-b34f-a07124adfe3e.png)

Only marker:
![image](https://user-images.githubusercontent.com/40357550/132844956-93871d37-3183-4c5d-8096-d5d79db457b7.png)

Is it possible to only have the message? Then the grid layout would need small adaptations

Reply:
Boclist.ShowEditModeValidationMarkers =false to switch off the marker
Boclist.DisableEditModeValidationMessages = true to switch off the message rendering

Reply:
This is all done in EditableRow.RenderSimpleColumnCellEditModeControl()